resume-buddy/src/main/java/com/google/sps/data/Reviewer.java

```diff
@@ -1,5 +1,13 @@
 package com.google.sps.data;
 
+enum Degree {
+  highschool,
+  associate,
+  bachelor,
+  master,
+  doctorate
+}
```

Comment:
Looks pretty good, just need to capitalize the enum values.

Enums in the Google style guide: https://google.github.io/styleguide/javaguide.html#s4.8.1-enum-classes

```
enum Degree {
  GED,
  ASSOCIATE,
  BACHELOR,
  MASTER,
  DOCTORATE
}
```

You'll probably want to put the enum in a file that is used by both the `Reviewer` and `Reviewee` classes.